### PR TITLE
feat: init extension with manifest and icons

### PR DESCRIPTION
**Pull request by frederickfogerty**

### CODE REVIEW

**Comment by frederickfogerty** [comment]
Current dependencies on/for this PR:
* next
  * **PR #173**  (this PR)
    * **PR #172** 

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

**Review by luqven — APPROVED** [review]
LGTM 👍🏼 

qq, where are these icons sourced from? FE repo? Just in case someone asks down the road.

File: src/extension/manifest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "imgix Salesforce B2C Commerce Cloud Extension",
+  "version": "0.0.1",
+  "manifest_version": 2,
+  "description": "This extension provides a Salesforce Commerce Cloud B2C integration with imgix.",
+  "homepage_url": "http://imgix.com",
```
Comment (luqven) [inline src/extension/manifest.json:6]: nit: should this be `https://`?
```suggestion
  "homepage_url": "https://imgix.com",
```

**Comment by frederickfogerty** [comment]
> qq, where are these icons sourced from? FE repo? Just in case someone asks down the road.

I got them from https://imgix.com/press

**Pull request merged**